az-cli/src/aro-preview/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
Do we need to support python 2?

Reply:
This is autogenerated

Reply:
> Do we need to support python 2?
Yes.

Reply:
@jim-minter @mjudeikis I'm pretty sure that Azure CLI only works with Python 3.6 or 3.7. And Python 2 is set to die in 15 days.